specs/Deployment/DeploymentAPI.md

```diff
@@ -164,6 +220,10 @@ namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime
         /// Checks the status of the WindowsAppRuntime of the current package and attempts to
         /// register any missing packages that can be registered.
         static DeploymentResult Initialize();
+
+        /// Checks the status of the WindowsAppRuntime of the current package and attempts to
+        /// register any missing packages that can be registered while applying the DeploymentOptions passed in
+        static DeploymentResult Initialize(Microsoft.Windows.ApplicationModel.WindowsAppRuntime::DeploymentOptions deploymentOptions);
```

Comment:
The method's new in 1.1 on a runtimeclass created in 1.0. Will need an APIcontract marker. See other comment